### ISSUE TITLE
Provided `Dockerfile` pulls incompatible version of OSQP

### ISSUE THREAD

**Issue by pwintz** [nicolapiccinelli/libmpc#19]
I used the provided Dockerfile to create a Docker container that includes all of the requirements for libmpc, but the version of OSQP appears to be incompatible. Downgrading to v0.6.3 appears to works. I suggest changing the provided Dockerfile to use this line for cloning the OSQP repo:
```
RUN git clone --depth 1 --branch v0.6.3 --recursive https://github.com/osqp/osqp /tmp/osqp \
    # .... Rest of the command here.
```

**Comment by nicolapiccinelli** [comment]
@pwintz yeah you are right, can you open a PR for that?

**Comment by nicolapiccinelli** [comment]
Included in 0.4.1 release
